HTML/about.html

```diff
@@ -136,28 +136,30 @@ <h2 class="cta-subHeader">Once submitted, your project will be viewable by your
     <p id="copy">Copyright 2019<!-- Find the code for the copyright symbol--></p>
   </footer>
 </body>
 <script src="/JS/github.js"></script>
+<script src="/JS/"></script>
```

Comment:
Looks like this is why the portfolio `eventListeners` weren't working.
```suggestion
<script src="/JS/portfolio.js"></script>
```